components/MainInfo.jsx

```diff
@@ -12,7 +12,41 @@ const MainInfo = ({ data }) => {
   return (
     <>
       <div className={styles.container}>
-        <div>
+        <div className={styles.leftContent}>
+          <h1 className={styles.city}>{data?.name}</h1>
+          <h1 className={styles.country}>{data?.sys?.country}</h1>
+          <p className={styles.icon}>
+            <img src={iconUrl} alt={`${description} icon`} />
+            {description
+              .toUpperCase()
+              .split(' ')
+              .map((part, i) => (
+                <Fragment key={i}>
```

Comment:
I think you can drop the Fragment and only use 
```js
< key={`description-item-${i}-${Math.random()}`}>
```

Please check if possible

Reply:
Deacuerdo @Israel-Laguan

Reply:
@Israel-Laguan it doesn't work 
![image](https://user-images.githubusercontent.com/61360270/202820333-f5ce44ff-488f-4476-b8e5-bdb6907a0dce.png)